examples/issue/components/IssueEditor.js

```diff
@@ -36,7 +36,7 @@ export default class IssueEditor extends React.Component {
     };
   }
 
-  onChange = (editorState, cb) => this.setState({ editorState }, cb);
+  onChange = (editorState) => this.setState({ editorState });
```

Comment:
Ah, this is mentioned in [`HOW_TO_CREATE_A_PLUGIN.md`](https://github.com/draft-js-plugins/draft-js-plugins/blob/master/HOW_TO_CREATE_A_PLUGIN.md):

<blockquote>
There is one difference compared to the original properties. All functions receive an additional argument. This argument is an object containing:
<br/>
<br/>
<pre>
// PluginFunctions
{
  getPlugins, // a function returning a list of all the plugins
  getProps, // a function returning a list of all the props pass into the Editor
  setEditorState, // a function to update the EditorState
  getEditorState, // a function to get the current EditorState
  getReadOnly, // a function returning of the Editor is set to readOnly
  setReadOnly, // a function which allows to set the Editor to readOnly
  getEditorRef, // a function to get the editor reference
}
</pre>
</blockquote>